test/integration/sonobuoy_integration_test.go

```diff
@@ -543,3 +539,30 @@ func TestExactOutput(t *testing.T) {
 		})
 	}
 }
+
+func TestOutputIncludes(t *testing.T) {
```

Comment:
Important to test it like this for clarity. TestExactOutput does what it says, tests all the exact output. This is just testing that some output includes....xyz.

Here we are testing an error message. This is important since we don't want to have golden files of our help output which would cause us to have to update golden files anytime we update unrelated flags, commands, help, etc.